tests/posix/timers/testcase.yaml

```
@@ -0,0 +1,25 @@
+common:
+  filter: not CONFIG_NATIVE_LIBC
+  tags:
+    - posix
+    - timers
+  # 1 tier0 platform per supported architecture
+  platform_key:
+    - arch
+    - simulation
+tests:
+  portability.posix.timers:
+    min_flash: 64
+    min_ram: 32
```

This is a copy-paste error that should be fixed in any of your pull requests.

It has already caused 3 regressions which I have patched.

Please try to catch this before committing.

```suggestion
  platform_key:
    - arch
    - simulation
  min_flash: 64
  min_ram: 32
tests:
  portability.posix.timers: {}
```